wegodang/users/utils.py

```diff
@@ -0,0 +1,23 @@
+import jwt
+
+from django.http  import JsonResponse
+
+from users.models import User
+from django.conf  import settings
+
+def login_required(func):
+    def wrapper(self,request,*args,**kwargs):
+        try:
+            client_token = request.headers.get("Authorization",None)
+            payload      = jwt.decode(client_token, settings.SECRET_KEY, settings.ALGORITHM)
+            user         = User.objects.get(id=payload['id'])
+            request.user = user
+
+        except jwt.exceptions.DecodeError:
+            return JsonResponse({'message' : ['INVALID_TOKEN'] }, status=400)
+
+        except User.DoesNotExist:
+            return JsonResponse({'message' : 'INVALID_USER'}, status=400)
```

Comment:
인증과 관련된 status code는 401을 많이 사용합니다.

```suggestion
        except jwt.exceptions.DecodeError:
            return JsonResponse({'message' : ['INVALID_TOKEN'] }, status=401)

        except User.DoesNotExist:
            return JsonResponse({'message' : 'INVALID_USER'}, status=401)
```

Reply:
수정했습니다 !